vllm/entrypoints/openai/api_server.py

```diff
@@ -189,6 +190,8 @@ async def create_chat_completion(request: ChatCompletionRequest,
         - function_call (Users should implement this by themselves)
         - logit_bias (to be supported by vLLM engine)
     """
+    if request.max_tokens is None:
+        request.max_tokens = max_model_len
```

Comment:
These lines wouldn't be needed in the scenario I proposed above.

Reply:
Thank you for the reminder. This solution is better. I will revise my code.